_data/people.yml

```diff
@@ -334,3 +334,14 @@
     - climate
     - software
     - GIS
+  
+ -first_name: Yu
+  last_name: Cheng
```

Comment:
Almost there. You need to indent everything by one more space. Look at the previous entry.